src/editor/parser/StateRender.js

```diff
@@ -531,14 +531,21 @@ class StateRender {
     return result
   }
 
-  loadImageAsync (src, alt, className, imageClass) {
+  loadImageAsync (imageInfo, alt, className, imageClass) {
+    const { src, isUnknownType } = imageInfo
     let id
     let isSuccess
 
+    console.log(src)
+
     if (!this.loadImageMap.has(src)) {
+      console.log('loadImageAsync: @load')
+
       id = getIdWithoutSet()
-      loadImage(src)
+      debounce(loadImage(src, isUnknownType)
```

Comment:
@Jocs I have a problem with `debounce` function here. It's called either multiple times after 1s timeout if user modify the url or never with `debounce(() = > { loadImage(...) [...] }, 1000)`. `debounce` should call the function only once as long as `debounce` continues to be invoked within 1s (or 0.5s here), right?

Need help to execute it only once within a 1 second.